plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py

```diff
@@ -213,6 +213,11 @@ def addManualDevice(self, address):
 
         self._checkManualDevice(address)
 
+    def _createMachineFromDiscoveredPrinter(self, key: str) -> None:
+        # TODO: This needs to be implemented. It's supposed to create a machine given a unique key as already discovered
+        # by this plugin.
+        pass
```

Comment:
We need to see who's going to implement this then...?

Reply:
This would also help in creating machines for printers that were found over the cloud API, and remove the need for having to be on the same local network to make that connection possible.